### PR TITLE
Add readme

### LINKED ISSUE #23: Quick Start
In addition to a brief description should describe how to get to the library for Node.js.

### PR DESCRIPTION
**Pull request by zxqfox**
Rel #22
Closes #23
Closes #97

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-bem-decl
-========
+# bem-decl
+
+Module to work with sets of BEM-entities (aka BEMDECL files).
```
Comment (tadatuta) [inline README.md]: BEM-entities -> BEM entities (here and bellow)

File: README.md [hunk 2]
```diff
@@ -8,17 +9,185 @@ bem-decl
...
-
 [david]:        https://david-dm.org/bem-sdk/bem-decl
-[david-img]:    http://img.shields.io/david/bem-sdk/bem-decl.svg?style=flat
+[david-img]:    https://img.shields.io/david/bem-sdk/bem-decl.svg
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en/) 4.x+
+
+## Installing
```
Comment (tadatuta) [inline README.md]: Installing -> Installation (to match with Usage bellow)

File: README.md [hunk 3]
```diff
@@ -8,17 +9,185 @@ bem-decl
...
+npm install --save bem-decl
+```
+
+## Usage
+
+```js
+const bemDecl = require('bem-decl');
+
+async function() {
+    const set1 = await bemDecl.load('set1.bemdecl.js');
```
Comment (tadatuta) [inline README.md]: isn't it to early for `async/await`?

File: README.md [hunk 4]
```diff
@@ -8,17 +9,185 @@ bem-decl
...
+Loads BEM-entities from a file in any format
+
+```js
+bemDecl.load('set1.bemdecl.js')
+    .then(decl => {
+        // Work with declaration
+    });
+```
+
+### `save(file: String, decl: BemCell[], opts: *): Promise<?>`
```
Comment (skad0) [inline README.md]: А нет такого в api же.

File: README.md [hunk 5]
```diff
@@ -8,17 +9,185 @@ bem-decl
...
+```js
+const decl = [
+    new BemCell({ entity: new BemEntityName({ block: 'button' }) })
+];
+bemDecl.save('set1.bemdecl.js', decl, { format: 'enb' });
+```
+
+### `merge(BemCell[], BemCell[], ...): BemCell[]`
+
+Merges many sets of BEM-entities into the one
```
Comment (tadatuta) [inline README.md]: ~~the~~

File: README.md [hunk 6]
```diff
@@ -8,17 +9,194 @@ bem-decl
...
+    new BemCell({ entity: new BemEntityName({ block: 'select' }) }),
+    new BemCell({ entity: new BemEntityName({ block: 'link' }) })
+];
+const decl2 = [
+    new BemCell({ entity: new BemEntityName({ block: 'link' }) })
+];
+const decl3 = [
+    new BemCell({ entity: new BemEntityName({ block: 'select' }) })
+];
+bemDecl.intersect(decl1, decl2, decl3).map(c => c.id);
```
Comment (skad0) [inline README.md]: subtract??

**Comment by zxqfox** [comment]
cc @godfreyd

**Comment by belozer** [comment]
Нужен README.ru.md

**Pull request merged**